public/index.html

```diff
@@ -0,0 +1,741 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Brand Marketing App</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="description" content="Project Week One">
+    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300,100,700' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Roboto+Condensed:400,700,300' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Raleway:400,100,300,500,600' rel='stylesheet' type='text/css'>
+    <link rel='stylesheet' href='css/stylesheet.css' type='text/css'>
+  </head>
+  <body>
+    <section class="home" id="home">
+      <section class="a1-home-header">
+        <section class="appbox-logo-section">
+          <img class= "appbox-logo" src="./img/appbox.png" alt="appbox logo">
+        </section>
+        <nav class="home-nav">
+          <ul>
+            <li><a class="home-nav-a" href="#home">HOME</a></li>
+            <li><a class="home-nav-a" href="#about">ABOUT</a></li>
+            <li><a class="home-nav-a" href="#features">FEATURES</a></li>
+            <li><a class="home-nav-a" href="#reviews">REVIEWS</a></li>
+            <li><a class="home-nav-a" href="#screens">SCREENS</a></li>
+            <li><a class="home-nav-a" href="#blog">BLOG</a></li>
+            <li><a class="home-nav-a" href="#support">SUPPORT</a></li>
+            <li class="icon-seek"></li>
+          </ul>
+        </nav>
+      </section>
+      <section class="a2-home-top">
+        <section class="home-h1-section">
+          <h1 class="home-h1 a2-h1">Best Way To Present Your<br> Beautiful Mobile App</h1>
+          <p class="home-p home-p-first-sentance">AppBox is full of awesome features. It's designed for</p>
+          <p class="home-p home-p-second-sentance"><strong>App Landing and App Store site</strong></p>
+          <section class="home-video">
+            <div class="icon-play home-video-border"></div>
+            <div><a class="home-intro-vid" href="">Watch the Intro Video</a></div>
+          </section>
+        </section>
+        <section class="home-phone-pic">
+          <img src="./img/overlayed-phones.png" class="blue-phone" alt="a blue iPhone on top of a green one">
+        </section>
+      </section>
+      <section class="a3-home-middle">
+        <svg class="slant" xmlns="http://www.w3.org/2000/svg" version="1.1" width="100%" height="100" viewBox="0 0 100 102" preserveAspectRatio="none">
+          <path d="M0 0 L0 100 L100 0 Z" />
+        </svg>
+        <h1 class="home-h1">GET IT FOR FREE</h1>
+        <p class="home-middle-p">Choose your native platform and download the app FREE!</p>
+        <ul class="home-google-ios">
+          <li><a class="submit-button" href="https://play.google.com/store?hl=en"><span class="icon-Googleplay"></span>  Google Play</a></li>
```

Comment:
A few things here:

1. While I appreciate that your buttons are finger-friendly (and, as such, appropriately mobile-first), at desktop resolutions you've moved pretty far from the design comp.  I'd reduce the `font-size` to about `1.4em` (16 * 1.4 = ~22px), and then update your `border-radius` to `18% / 50%`, and that gets you closer. Be mindful of the line-heights and the way your icons are sitting as well.
2. `home-google-ios` is a pretty specific classname that can quickly become outdated. When naming something like this, think of the purpose these buttons serve. Rather than `a3-home-middle`, I would have called this whole section the `call-to-action`; the buttons could be `store-link`s, or even just anchors that live inside of an `actions` container (in this case, your list)
3. These links aren't really submitting anything, so the `submit-` prefix is a little inaccurate.